components/ws-manager-bridge/src/meta-instance-controller.ts

```diff
@@ -31,21 +31,21 @@ export class MetaInstanceController {
             try {
                 log.debug(logContext, 'MetaInstanceController: Checking for workspaces to stop');
 
+                const now = Date.now();
                 const creationTime = new Date(instance.latestInstance.creationTime).getTime();
-                const preparingKillTime = creationTime + (this.config.timeouts.preparingPhaseSeconds * 1000);
-                const unknownKillTime = creationTime + (this.config.timeouts.unknownPhaseSeconds * 1000);
-                const exceededPreparingTime = Date.now() >= preparingKillTime;
-                const exceededUnknownTime = Date.now() >= unknownKillTime;
-                const currentState = instance.latestInstance.status.phase;
+                const timedOutInPreparing = now >= creationTime + this.config.timeouts.preparingPhaseSeconds * 1000;
```

Comment:
Will the `* 1000` bit target the correct bit of logic? I'm always concerned it will do the `(creationTime + number) * 1000` meaning it'll never trigger

Reply:
:+1: